libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -546,12 +561,23 @@ void AP_Baro::init(void)
 #else
         ADD_BACKEND(AP_Baro_MS56XX::probe(*this,
                                           std::move(hal.i2c_mgr->get_device(_ext_bus, HAL_BARO_MS5611_I2C_ADDR))));
+
+ #if CONFIG_HAL_BOARD == HAL_BOARD_F4LIGHT // we don't know which baro user will solder
```

Comment:
Maybe not needed for now, but we should add an external driver enable mask parameter here; the issue is that people might not want to enable the MS56XX driver as it conflicts with other device types (as @WickedShell said). Also these baros below might be used in other boards as well.

Reply:
I don't affect other boards, I need to fix user's issue